README.md

```diff
@@ -1,82 +1,86 @@
 # CondeNast/renovate-config
 
-This is a collection of [GitHub-hosted presets](https://docs.renovatebot.com/config-presets/#github-hosted-presets) for Renovate dependency bot.
+This is a collection of [GitHub-hosted presets](https://docs.renovatebot.com/config-presets/#github-hosted-presets) for Renovate dependency bot. 🤖
 
 ## Usage
 
-renovate.json
+In your `renovate.json`:
 
 ```
 {
   "extends": [
-    "config:base",
-    "github>CondeNast/renovate-config:groupMinorUpdates(dependencies)"
-    "github>CondeNast/renovate-config:forceCondenastUpdates",
+    "github>CondeNast/renovate-config",
+    "github>CondeNast/renovate-config:forceCondenastUpdates"
+  ]
+}
+```
+
+The `CondeNast/renovate-config` preset should go at the start of the `extends` array.
+
+The `CondeNast/renovate-config:forceCondenastUpdates` preset should go at the end of your `extends` array to ensure that PRs for Condé Nast package updates are created as soon as possible, to bypass any rate limiting and schedules.
+
+If you would like to [auto-merge](https://docs.renovatebot.com/configuration-options/#automerge) minor upgrades, here is a recommended configuration for doing that:
+
+```
+{
+  "extends": [
+    "github>CondeNast/renovate-config",
     "github>CondeNast/renovate-config:automergeMinor",
-    "github>CondeNast/renovate-config:condenastStabilityDays",
     "github>CondeNast/renovate-config:dontAutomergeMajor(help wanted)",
     "github>CondeNast/renovate-config:dontAutomergeNode(help wanted)",
     "github>CondeNast/renovate-config:dontAutomergeTestFrameworks(help wanted)",
     "github>CondeNast/renovate-config:dontAutomergeCIDependencies(help wanted)",
+    "github>CondeNast/renovate-config:condenastStabilityDays",
+    "github>CondeNast/renovate-config:forceCondenastUpdates"
   ]
 }
 ```
 
-Alternatively:
+You can also use any individual **semantic presets** without using the base preset. Here's a configuration that groups things but doesn't auto-merge them:
 
 ```
 {
   "extends": [
     "config:base",
-    "github>CondeNast/renovate-config"
+    "github>CondeNast/renovate-config:groupMinorUpdates(dependencies)",
+    "github>CondeNast/renovate-config:groupCIDependencyUpdates",
+    "github>CondeNast/renovate-config:groupLinterUpdates",
+    "github>CondeNast/renovate-config:groupNodeUpdates",
+    "github>CondeNast/renovate-config:forceCondenastUpdates"
   ]
 }
 ```
 
-## Presets
```

Comment:
Removed the descriptions for each preset from the readme because this was hard to maintain